packages/prejournal/README.md

```diff
@@ -1,6 +1,29 @@
-⚠️ This package is an untested sketch for [prejournal](https://prejournal.org/) timesheets  sync integration.
+# timeld - Prejournal integration
 
-## API docs
+This packages supports live update of timesheet entries:
+- _from_ **timeld**
+- _to_ [prejournal](https://prejournal.org/).
+
+## installation & configuration
+
+This integration package is included in the gateway.
+
+The integration requires the following configuration:
+- `user`: Prejournal account (user) name (required)
+- `key`: Prejournal account key (required)
+- `api`: API URL e.g. `https://time.pondersource.com/v1/` (required)
+- `client`: default timesheet "client" (for whom the work is being done; required)
+
+Configuration defaults may already be included in the gateway. Timesheet-specific configuration can also be included when adding the integration, see below.
```

Comment:
1. As mentioned in earlier comment on another README.md in this review, would recommend capitalising "Gateway"; and
2. Replace "integration" with "connector".